src/pages/patientView/structuralVariant/StructuralVariantTableWrapper.tsx

```diff
@@ -474,12 +474,27 @@ export default class PatientViewStructuralVariantTable extends React.Component<
             this.columns,
         ],
         render: () => {
+            const isProfiled = this.props.store.structuralVariantProfile.result
```

Comment:
we should work on this logic here.  it's quite brittle and can be made more tolerant by using something called "optional chaining".   i see that this is following the pattern above so we can improve there too.